README.md

```diff
@@ -1,7 +1,5 @@
 # Node client for Usabilla API
 
-[![Build Status](https://travis-ci.org/usabilla/api-js-node.svg?branch=master)](https://travis-ci.org/usabilla/api-js-node?branch=master) [![Coverage Status](https://coveralls.io/repos/github/usabilla/api-js-node/badge.svg?branch=master)](https://coveralls.io/github/usabilla/api-js-node?branch=master)
```

Comment:
We should still keep the build and coverage badges, right? It is just that the build should point to CircleCI one now.